pkg/cloud/metadata/metadata.go

```diff
@@ -136,11 +136,24 @@ func newImmutableProvider(provider MetadataProvider, name string) *immutableProv
 }
 
 func NewMetadata() *Metadata {
-	return &Metadata{
+	defaultMetadata := &Metadata{
 		providers: []MetadataProvider{
 			newImmutableProvider(&ENVMetadata{}, "env"),
 		},
 	}
+	data, err := os.ReadFile(LingjunConfigFile)
+	if err != nil {
+		klog.V(1).InfoS("can't read lingjun metadata file, use env metadata only", "file", LingjunConfigFile, "error", err)
```

Comment:
Maybe we should log "no such file" error at V(3), and other errors with klog.Error